packages/react-paypal-js/src/v6/context/PayPalProviderContext.tsx

```diff
@@ -46,4 +46,7 @@ export function instanceReducer(
     }
 }
 
-export const PayPalContext = createContext<PayPalContextState | null>(null);
+export const PayPalContext = createContext<PayPalContextValue>({
```

Comment:
This wasn't quite the change I was looking for, but I think these types are a little bit cleaner now. `PayPalContextValue` inherits from `PayPalState` and the context will never be `null`.